### PR TITLE
Fix docs on `--host[-ipv6]-addr` options

### LINKED ISSUE #2352: [BUG] - IPV6 addresses fail to parse
**Internal/External**
*External*   otherwise.

**Area**
*Other*   

**Summary**
Add an IPv6 address as external address. Get an error.

**Steps to reproduce**
I run this:
```
cardano-node run \
   --socket-path /home/cardano/configuration/socket \
   --topology /home/cardano/configuration/mainnet-topology.json \
   --database-path /home/cardano/configuration/db \
   --host-addr  2601:640:8880:375b:2c0:caff:fe47:9d63 \
   --port 3001 \
   --config /home/cardano/configuration/mainnet-config.json
```
Get error
```
option --host-addr: Failed to parse: 2601:640:8880:375b:2c0:caff:fe47:9d63
```
**Expected behavior**
cardano-node should start up and bind to the given address

**System info (please complete the following information):**
- OS: debian 
- Version 10.7
- Node version cardano-node-1.25.0-linux.tar.gz

**Additional context**
Documentation explicitly states that host-addr can be an IPv4 or IPv6 address. 
https://docs.cardano.org/projects/cardano-node/en/latest/
"--host-addr - Optionally specify your node’s IPv4 or IPv6 address."

### PR DESCRIPTION
**Pull request by cblp**
Make them consistent with the real behaviour.

Fix #2352

### CODE REVIEW

**Comment by cblp** [comment]
@karknu please review

**Comment by Jimbo4350** [comment]
@cblp It's a good idea to include the issue you are solving in the main comment of this PR. Anything like "Closes #2352" would suffice

File: cardano-node/src/Cardano/Node/Parsers.hs [hunk 1]
```diff
@@ -100,24 +100,34 @@ parseHostIPv4Addr =
...
+      ". Use --host-ipv6-addr to specify IPv6 adddress")
+    (Right . NodeHostIPv4Address)
+    (readMaybe str)
 
 parseNodeHostIPv6Address :: String -> Either String NodeHostIPv6Address
 parseNodeHostIPv6Address str =
-   maybe (Left $ "Failed to parse: " ++ str) (Right . NodeHostIPv6Address) $ readMaybe str
+  maybe
+    (Left $
+      "Failed to parse: " ++ str ++
```
Comment (Jimbo4350) [inline cardano-node/src/Cardano/Node/Parsers.hs:127]: Does this solve the issue the user was having? I.e does the IPv6 address "2601:640:8880:375b:2c0:caff:fe47:9d63" successfully parse now?

Comment (Jimbo4350) [reply cardano-node/src/Cardano/Node/Parsers.hs:127]: Also I think you mixed up your error messages.

**Review by Jimbo4350 — CHANGES_REQUESTED** [review]

File: cardano-node/src/Cardano/Node/Parsers.hs (hunk 1 above)
Comment (cblp) [reply cardano-node/src/Cardano/Node/Parsers.hs:127]:
```
$ cardano-node run --host-addr 2601:640:8880:375b:2c0:caff:fe47:9d63
option --host-addr: Failed to parse: 2601:640:8880:375b:2c0:caff:fe47:9d63. Use --host-ipv6-addr to specify IPv6 adddress
```

Comment (cblp) [reply cardano-node/src/Cardano/Node/Parsers.hs:127]: I don't see anything mixed up. V4 error suggests V6 and vice versa.

**Comment by cblp** [comment]
@Jimbo4350 I put the bug report reference to the commit and forgot to double it in the PR. Now fixed.

**Review by Jimbo4350 — APPROVED** [review]
LGTM! I would wait for approval from @karknu as well.

**Review by karknu — APPROVED** [review]
LGTM.

**Pull request merged**